src/controllers/ProductsController.js

```diff
@@ -415,7 +415,6 @@ class ProductsController extends Controller {
 
     const devices = (await this._deviceAttributeRepository.getManyFromIDs(
       deviceIDs,
-      this.user.id,
```

Comment:
ah..ok, in the future, we will change the line to `this._permissionManager`.getManyFromIDs`.